### PR TITLE
Test/add sections basic tests

### PR DESCRIPTION
**Pull request by molnosikristof**

### CODE REVIEW

File: e2e/test/section-tests/section-test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  it('should check the section ' + DANGER_ZONE, () => {
+    browser.url('/sections');
+    assert(browser.isExisting(DANGER_ZONE_SELECTOR), 'Danger zone is not existing in the DOM');
+    assert(takeScreenShotOfElement(DANGER_ZONE_SELECTOR,
+      {firefoxTolerance: 4.5, defaultTolerance: 3, ignoreComparison: false}),
+       'Added tags input is not similar to the reference');
+  });
+
+  it.only('should check the section ' + DANGER_ZONE_BUTTON_MOUSEOVER, () => {
```
Comment (gazdagergo) [inline e2e/test/section-tests/section-test.js]: `only` should be deleted

File: e2e/test/section-tests/section-test.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    browser.url('/sections');
+    const isItSauce = (process.env.E2EPROFILE || '').includes('sauce');
```
Comment (oroce) [inline e2e/test/section-tests/section-test.js]: ahhhh, is it the best solution?

Comment (molnosikristof) [reply e2e/test/section-tests/section-test.js]: You mean for the separation if you are run locally or on sauce? Maybe I can use the browser.desiredCapabilities.platfrom.

Comment (oroce) [reply e2e/test/section-tests/section-test.js]: yeah that sounds more reasonable

Comment (molnosikristof) [reply e2e/test/section-tests/section-test.js]: I've added extra parameters, that hides from the test writer this step.

File: e2e/test/section-tests/section-test.js (hunk 1 above)
Comment (ruffle1986) [reply e2e/test/section-tests/section-test.js]: nice catch! hawkeye! 🍻

**Pull request merged**